### PR TITLE
Fix/misc styling

### PR DESCRIPTION
**Pull request by rdtho2525**
# Selector FE PR

## What does this merge do? Check all that apply.
- [ ] Adds a new feature
- [ ] Improves an existing feature
- [X] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## Summary of changes
 + Style search form
 + Adjust breakpoint for nav order
 + Style hamburger icon
 + Replace Discogs svg w/ a white one
 + Change text color for Details
 + Adjust for contrast error on genre elements
 + Apply styling to error messages

## Notes for reviewer (any known bugs, areas of interest, etc.)
- Not directly pertaining to this PR

## Next Steps (Future Iterations)
- Details testing
- Presentation

### CODE REVIEW

File: src/Components/Search/Search.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ const Search = ({ setSearchArtist }) => {
     } 
   
     if (type === 'album') {
+      setArtist(userQuery)
+    } else {
       event.preventDefault()
       history.push(`/${userQuery}`)
```
Comment (Max9545) [inline src/Components/Search/Search.js]: History Object could possibly be utilized more

File: src/Components/Details/Details.js [hunk 2]
```diff
@@ -47,10 +48,12 @@ const Details = ({
...
-  if (loading1) return <p>Loading...</p>; 
-  if (error1) return <p>Error :(</p>;
-  if (loading2) return <p>Loading...</p>; 
-  if (error2) return <p>Error :(</p>;
+  if (loading1 || loading2) return (
+    <div className='card-container' data-cy='card-container'>
+      <Skeleton animation="wave" height={675} width={878} />
+    </div>
+  )
+  if (error1 && error2) return <h2 className="details_error">⚠️ We're sorry - something went wrong! Please try again later.</h2>;
```
Comment (monshan) [inline src/Components/Details/Details.js]: Let's just change this to error1 || error2 to ensure coverage for both

**Review by monshan — APPROVED** [review]

**Comment by monshan** [comment]
Giving approval to merge to main due to conflicts on github allowance for rebase and merge

**Comment by rdtho2525** [comment]
In accordance with @monshan, I will merge this PR as I have the most up-to-date edition with resolved conflicts.

**Pull request merged**